app/main.py

```diff
@@ -1,3 +1,61 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
```

Comment:
It would be a good idea to add a docstring to your class to explain what it represents and how it should be used.